spec.bs

```diff
@@ -1260,8 +1248,35 @@ an [=ordered map=] whose [=map/keys=] are [=strings=] and whose [=map/values=] a
     1. Set |global|'s
       [=InterestGroupBiddingScriptRunnerGlobalScope/is component auction=] to |isComponentAuction|.
     1. Set |global|'s [=InterestGroupBiddingScriptRunnerGlobalScope/interest group=] to |ig|.
-    1. Return the result of [=evaluating a script=] with |global|, |script|, "`generateBid`",
+    1. Let |startTime| be the current time.
+    1. Let |result| be the result of [=evaluating a script=] with |global|, |script|, "`generateBid`",
       |arguments|, and |timeout|.
+    1. Let |duration| be the current time minus |startTime| in milliseconds.
+    1. If |global|'s [=InterestGroupBiddingScriptRunnerGlobalScope/priority=] is not null:
+      1. Set |ig|'s [=interest group/priority=] to |global|'s
+        [=InterestGroupBiddingScriptRunnerGlobalScope/priority=].
+      1. Store |ig| in the browser’s [=interest group set=].

```

Comment:
IIUC, |ig| came from the interest group set in the first place, via https://pr-preview.s3.amazonaws.com/WICG/turtledove/pull/605.html#build-bid-generators-map, so I think you mean to update the interest group instead of storing a new one. I see that https://pr-preview.s3.amazonaws.com/WICG/turtledove/pull/605.html#interest-group-update uses [Replace](https://infra.spec.whatwg.org/#list-replace) to do that, and you could do the same here using the owner and name fields.

Reply:
Note that I didn't use the owner and name fields, like was done in the interest group update code.  Let me know if I should use them somehow.

Reply:
BTW I'm fixing the other call site in https://github.com/WICG/turtledove/pull/623